web/templates/navbar.hbs

```diff
@@ -55,7 +55,11 @@
             </div>
             <div id="personal-menu" class="hidden-for-spectators navbar-item">
                 <a class="header-button tippy-zulip-delayed-tooltip" tabindex="0" role="button" data-tooltip-template-id="personal-menu-tooltip-template">
-                    <img class="header-button-avatar" src="{{user_avatar}}"/>
+                    <div class="header-button-avatar">
+                        <!-- If you add attributes to this img tag, update the list of
+                        expected tags in ui_util.restore_ublockorigin_hidden_img. -->
```

Comment:
I know this is kind of confusing and ideally it would be enforced with a test (or a type constraint if that were available). But I think it would require e2e test and it seems you like to keep those tests basic so they run fast. This is a rather niche thing. As I've written `restore_ublockorigin_hidden_img`, the worst thing that could happen is that a user who blocks gravatar with uBlock Origin would have to reload the page to see their newly uploaded avatar.

Reply:
Hmm. Perhaps it's possible to inspect the real hbs templates with zjquery? I'm a little confused what it can actually do. Or otherwise, I could probably just do a crude text-based check to make sure that no weird attributes are added to the img tag.